### ISSUE TITLE
Script train_image_classifier.py has no option use_nesterov in the MomentumOptimizer

### ISSUE THREAD

**Issue by parag2489** [tensorflow/tensorflow#17024]
**Have I written custom code:** No
**OS Platform and Distribution:** Ubuntu 17.04
**TensorFlow installed from:** Pip
**TensorFlow version:** 1.4.0
**Bazel version:** N/A
**CUDA/cuDNN version:** CUDA 8.0.61, CUDNN v6
**GPU model and memory:** Tesla K80
**Exact command to reproduce:** N/A

In Tensorflow, the `MomentumOptimizer` has `use_nesterov` disabled by default. In the generic classifier training script - `train_image_classifier.py` - the flags do not have `use_nesterov` option (see [here](https://github.com/tensorflow/models/blob/4c05414826e87f3b8ef0534862748e4b7fcd1ec7/research/slim/train_image_classifier.py#L297) and [here](https://github.com/tensorflow/models/blob/4c05414826e87f3b8ef0534862748e4b7fcd1ec7/research/slim/train_image_classifier.py#L116)), which makes it real easy to miss the fact that it is not being used. I think an additional flag should be introduced in the optimizer section of `train_image_classifier.py`.

P.S. Why was the decision taken to disable `use_nesterov`? I tried searching online if there were any disadvantages associated with Nesterov momentum but did not come up with much. [The paper referenced in Tensorflow documentation](http://proceedings.mlr.press/v28/sutskever13.pdf) only encourages the use of Nesterov momentum versus classical momentum. Am I missing something?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
We are closing this issue for now due to lack of activity. Please comment if this is still an issue for you. Thanks!
